code/LeftAndMain.php

```diff
@@ -784,7 +784,7 @@ protected function init()
             }
         }
 
-        $this->extend('init');
+        $this->extend('onInit');
```

Comment:
Intentionally not changing to `onAfterInit` because this isn't at the end of the method.